### ISSUE TITLE
Current status callout

### ISSUE THREAD

**Issue by pennyandrews** [vegaprotocol/vega.xyz#188]
## Story

Design and build call out for current status "The Vega network is live, you can't trade yet..." (on use vega page, key concepts)

## Tasks

- [x] Brief designer
- [x] Design
- [x] Review how this might work if it gets stacked with the translation banner
- [x] Team review
- [x] Implementation
- [ ] Team testing
- [ ] Share with vega
- [ ] Release

## Sketch

![image](https://user-images.githubusercontent.com/47143694/190702007-5080a398-335d-472d-a557-07dabb67d528.png)

**Comment by pennyandrews** [comment]
Joëlle:

2 attempts [here](https://www.figma.com/file/qgCd4r61Z1r3tUkxKehhmu/Website-Design-Updates?node-id=243%3A17863) → 

Inline callout: follows callout pattern with a button CTA rather than a link
Notification style banner: sits under the nav, feels like it gets a bit "stripy"

Thoughts or any better suggestions? I don't mind the callout option in the end... 

(Ignore how the header illustration is cropped / covered in some options)

**Comment by pennyandrews** [comment]
Joëlle:

Tried two approaches:

[Callout with a button](https://www.figma.com/file/qgCd4r61Z1r3tUkxKehhmu/Website-Design-Updates?node-id=243%3A17863)
[Notification style banner under the primary nav](https://www.figma.com/file/qgCd4r61Z1r3tUkxKehhmu/Website-Design-Updates?node-id=243%3A19907)

Applied to Key Concepts and Use the Network pages. Feels like a) the callout is clearer and more actionable and b) that we need a version of the callout "with button" for instances like this where a link feels like it's not strong enough for the CTA

Let me know your thoughts!

**Comment by pennyandrews** [comment]
Calum:

I don't mind the callout option either 🙂

**Comment by pennyandrews** [comment]
Joëlle:

Attached

The double callout - not sure about it? I wonder if this translation banner is actually a full width type banner that sits flush under the nav (far right option) as it is quite important and in purple feels less "stripey" to me that it did when I tried the callout option like this in grey?

Could then use this full width pattern for any urgent warning type messages, eg. if something is down for example?

Regardless, in the far left and middle options we should probably align the font sizes as it's noticeably different and feel inconsistent when you see the two callout types alongside one another. As well as the border radius (let's agree on what this is!)

In the far right option, I removed the heading and made it all 16pts with a much skinner height. Thinking could use bold if needing to pull out bits of text. Just some thoughts, let me know what you think.

<img width="2992" alt="Screenshot 2022-08-16 at 14 43 52 (1)" src="https://user-images.githubusercontent.com/47143694/190702274-34fb2682-5baf-46c9-afa0-51c66d6ede62.png">

**Comment by pennyandrews** [comment]
David:

You mean I have to commit to a corner radius? 👀

😂😂😂 FIIIIIINE.

I think the full-width option works pretty well. Am thinking if it's aligning more with the nav bar then we should align the text with this too (rather than sit it on the main grid).

Let's drop the outline stroke on this bar as it makes the edge of the purple look a little blurry.

Agreed on the consistent text sizes... I was being self-indulgent (as always!).

I guess this example is the worst case scenario and it looks tidy.

Side note: I'm aware I have pinched the planet design/layout from this page for the new landing page design 🙃 - I'll make sure we have an alternative for the Key Concepts page, probably something not as "designed".

Thanks [@Joëlle Snaith](https://app.asana.com/0/1202216849708246/list)!

**Comment by pennyandrews** [comment]
Joëlle:

😂🙃

Good shout about the border! 

I've tried centre aligning (was that what you meant?) and left aligning the content to the nav items - I guess centred is probably the safest implementation option? Found the right icon started fighting with the dark / light mode icon in the primary nav so tried it on the left... and without... could also try reduce the overall height by a couple of pixels but might start feeling too tight. LMK...

Agree this is a worst case scenario! 

Noted re planet pinching 🤣🪐
